debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -440,6 +440,22 @@ private boolean schemaChanged(List<ReplicationMessage.Column> columns, Table tab
                                 incomingType);
                     return true;
                 }
+                if (metadataInMessage) {
```

Comment:
To be sure, we get length and scale from the (latest) wal2json and decoderbufs, correct?

Reply:
Yes, from the latest - unfrotunately we can remove it only after RDS will move forward with `wal2json` version